combiner/main.go

```
@@ -16,26 +16,20 @@ func main() {
 	var tconf tdns.Config
 	// var mconf music.Config
 
-	tconf.App.Mode = "combiner"
-	tconf.App.Version = appVersion
-	tconf.App.Name = appName
-	tconf.App.Date = appDate
+	tdns.Globals.App.Type = tdns.AppTypeCombiner
+	tdns.Globals.App.Version = tdns.Globals.App.Version
```

golangci-lint (1.62.2)

assign: self-assignment of tdns.Globals.App.Version to tdns.Globals.App.Version

(govet)